vita3k/interface.cpp

```diff
@@ -574,6 +574,7 @@ bool handle_events(EmuEnvState &emuenv, GuiState &gui) {
             emuenv.kernel.exit_delete_all_threads();
             emuenv.gxm.display_queue.abort();
             emuenv.display.abort.exchange(true);
```

Comment:
btw could you change this line to 
`emuenv.display.abort = true;`
It would look cleaner

Reply:
yes, of course